packages/babel-compat-data/scripts/data/corejs2-built-in-features.js

```diff
@@ -73,8 +73,8 @@ module.exports = {
   "es6.array.of": "Array static methods / Array.of",
   "es6.array.reduce": "Array methods / Array.prototype.reduce",
   "es6.array.reduce-right": "Array methods / Array.prototype.reduceRight",
-  // compat-table missing babel6 mapping
```

Comment:
Is this still a valid reason? I'm seeing a few more `compat-table missing babel6 mapping` here

Reply:
Uh no, we can add all of them back if they are in compat-table.

Reply:
Noted, let me add them back then